### ISSUE TITLE
[3.4] Pagination is missing on overview pages in backend.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#6999]

**Comment by bobdenotter** [comment]
Tracing it, here is where it diverges between 3.3 and 3.4: 

https://github.com/bolt/bolt/blob/3.3/src/Twig/Runtime/RecordRuntime.php#L243

```
        if ($this->pagerManager->isEmptyPager()) {
            // nothing to page.
            return '';
        }
```

On 3.4 it's `true`, and it returns.

**Comment by GawainLynch** [comment]
For reference, that is behaving correctly.

In 3.4 the listings are no longer using the old storage layer, and [this equivalent functionality](https://github.com/bolt/bolt/blob/3.4/src/Legacy/Storage.php#L1668-L1682) needs to find a new home

**Comment by bobdenotter** [comment]
Closed by #7014
